builder/bundles.go

```diff
@@ -586,6 +597,15 @@ func buildFullChroot(cfg *buildBundlesConfig, b *Builder, set *bundleSet, packag
 	return nil
 }
 
+func writeBundleInfoPretty(bundle *bundle, path string) error {
+	b, err := json.MarshalIndent(*bundle, "", "	")
```

Comment:
Minor suggestion, use \t rather than a literal tab character. After your comment about the size of the file I saw this line in the diff and thought "I wonder if that is a number of spaces, and that makes the size so large?"

Reply:
Ah, yes I'll make it a \t

Reply:
done.